src/common/components/hive-engine-chart/index.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import { Theme } from "../../store/global/types";
+import ReactHighcharts from "react-highcharts"
+import moment from 'moment';
+import { _t } from '../../i18n';
+
+export const HiveEngineChart = (props: any) => {
+const { items, theme } = props
+// console.log(items)
+
+const [close, setClose]: any = useState([])
+const [timeStamp, setTimeStamp]: any = useState([]);
+const [tokenSymbol, setTokenSymbol] = useState("SWAP.HIVE")
+const [prices, setPrices]: any = useState([])
+
+useEffect(() => {
+  const getMarketData = async (symbol: any) => {
+      const { data: history } = await axios.get(
+          `https://info-api.tribaldex.com/market/ohlcv`,
```

Comment:
Can we set all hive-engine servers inside servers.json or create he-servers.json inside constants folder? So we don't hard code all separately but from single place when issue occurs we can replace from single place, not from all random components.